File: udemy/java/matriz3x3/src/matriz3x3/Matriz3x3.java
package matriz3x3;
import javax.swing.JOptionPane;
public class Matriz3x3 {
	public static void main(String[] args) {
		int vet[][]= new int [3][3];
		String mensagem = "";
		for (int l = 0; l <= 2; l++) {
			for(int c = 0; c <= 2; c++) {
				vet[l][c] = Integer.parseInt(JOptionPane.showInputDialog("informe o valor do vetor na posição ["+(l+1)+", "+(c+1)+"]: "));
				mensagem = mensagem + "O valor do vetor na posição ["+(l+1)+", "+(c+1)+"] é:  "+vet[l][c]+"\n";
			}
		}
		JOptionPane.showMessageDialog(null, mensagem);
		
	}

}

File: udemy/pascal/projetos/matriz3x3.pas
program matriz3x3;
var 
    vet : array[1..3,1..3] of Integer;
    l,c : Integer;
begin
    for l := 1 to 3 do
        begin
            for c := 1 to 3 do
            begin
                write('informe o valor do vetor na posição [',l,', ',c,']: ');
                read(vet[l,c]);
            end;
        end;
    for l := 1 to 3 do
        begin
            for c := 1 to 3 do 
                begin
                    writeln('o valor do vetor na posição [',l,', ',c,'] é: ', vet[l,c]);
                end;
        end;
   
end.
